pkg/scheduler/schedule_one_test.go

```diff
@@ -837,21 +843,30 @@ func TestSchedulerScheduleOne(t *testing.T) {
 				}
 				sched.ScheduleOne(ctx)
 				<-called
-				if e, a := item.expectAssumedPod, gotAssumedPod; !reflect.DeepEqual(e, a) {
-					t.Errorf("assumed pod: wanted %v, got %v", e, a)
+				if diff := cmp.Diff(item.expectAssumedPod, gotAssumedPod, cmpopts.IgnoreFields(v1.PodCondition{}, "LastTransitionTime")); diff != "" {
+					t.Errorf("assumed pod: wanted %v, got %v", item.expectAssumedPod, gotAssumedPod)
 				}
-				if e, a := item.expectErrorPod, gotPod; !reflect.DeepEqual(e, a) {
-					t.Errorf("error pod: wanted %v, got %v", e, a)
+				if diff := cmp.Diff(item.expectErrorPod, gotPod, cmpopts.IgnoreFields(v1.PodCondition{}, "LastTransitionTime")); diff != "" {
+					t.Errorf("error pod: wanted %v, got %v", item.expectErrorPod, gotPod)
 				}
-				if e, a := item.expectForgetPod, gotForgetPod; !reflect.DeepEqual(e, a) {
-					t.Errorf("forget pod: wanted %v, got %v", e, a)
+				if diff := cmp.Diff(item.expectForgetPod, gotForgetPod, cmpopts.IgnoreFields(v1.PodCondition{}, "LastTransitionTime")); diff != "" {
+					t.Errorf("forget pod: wanted %v, got %v", item.expectForgetPod, gotForgetPod)
 				}
 				if e, a := item.expectError, gotError; !reflect.DeepEqual(e, a) {
 					t.Errorf("error: wanted %v, got %v", e, a)
 				}
 				if diff := cmp.Diff(item.expectBind, gotBinding); diff != "" {
 					t.Errorf("got binding diff (-want, +got): %s", diff)
 				}
+				if item.eventReason == "Scheduled" {
+					updatedPod, err := client.CoreV1().Pods(item.sendPod.Namespace).Get(ctx, item.sendPod.Name, metav1.GetOptions{})
+					if err != nil {
+						t.Fatal(err)
+					}
+					if diff := cmp.Diff(updatedPod.Status.Conditions, podStatusConditions, cmpopts.IgnoreFields(v1.PodCondition{}, "LastTransitionTime")); diff != "" {
```

Comment:
```suggestion
					if diff := cmp.Diff(podStatusConditions, updatedPod.Status.Conditions, cmpopts.IgnoreFields(v1.PodCondition{}, "LastTransitionTime")); diff != "" {
					t.Errorf("unexpected conditions (-want, +got):\n%s", diff)
```

Print diff when there is. Also do this for above lines.